{
 "cells": [
  {
   "cell_type": "code",
   "execution_count": 1,
   "id": "bb3f1cc4",
   "metadata": {},
   "outputs": [],
   "source": [
    "from collections import Counter, defaultdict\n",
    "from pathlib import Path\n",
    "\n",
    "import matplotlib.pyplot as plt\n",
    "import pandas as pd\n",
    "import seaborn as sns\n",
    "from matplotlib_inline.backend_inline import set_matplotlib_formats\n",
    "from tabulate import tabulate\n",
    "\n",
    "set_matplotlib_formats(\"svg\")"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 2,
   "id": "aec0b15e",
   "metadata": {},
   "outputs": [],
   "source": [
    "here = Path().resolve()\n",
    "affiliations_path = here.joinpath(\"affiliations.tsv\")\n",
    "authors_path = here.joinpath(\"authors.tsv\")\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 3,
   "id": "ac4ac23f",
   "metadata": {},
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "<>:155: SyntaxWarning: 'tuple' object is not callable; perhaps you missed a comma?\n",
      "<>:155: SyntaxWarning: 'tuple' object is not callable; perhaps you missed a comma?\n",
      "/var/folders/f8/snw3tlcs0y3bmnq5gqg26b5h0000gn/T/ipykernel_53320/2181873963.py:155: SyntaxWarning: 'tuple' object is not callable; perhaps you missed a comma?\n",
      "  (204, 391) #  Rahel Paloots\n"
     ]
    },
    {
     "ename": "TypeError",
     "evalue": "'tuple' object is not callable",
     "output_type": "error",
     "traceback": [
      "\u001b[0;31m---------------------------------------------------------------------------\u001b[0m",
      "\u001b[0;31mTypeError\u001b[0m                                 Traceback (most recent call last)",
      "Cell \u001b[0;32mIn [3], line 155\u001b[0m\n\u001b[1;32m      1\u001b[0m person_rename \u001b[38;5;241m=\u001b[39m {\n\u001b[1;32m      2\u001b[0m     \u001b[38;5;124m\"\u001b[39m\u001b[38;5;124mCristina Casals Casas\u001b[39m\u001b[38;5;124m\"\u001b[39m: \u001b[38;5;124m\"\u001b[39m\u001b[38;5;124mCristina Casals-Casas\u001b[39m\u001b[38;5;124m\"\u001b[39m,\n\u001b[1;32m      3\u001b[0m     \u001b[38;5;124m\"\u001b[39m\u001b[38;5;124mChris Mungall\u001b[39m\u001b[38;5;124m\"\u001b[39m: \u001b[38;5;124m\"\u001b[39m\u001b[38;5;124mChristopher J. Mungall\u001b[39m\u001b[38;5;124m\"\u001b[39m,\n\u001b[0;32m   (...)\u001b[0m\n\u001b[1;32m     20\u001b[0m     \u001b[38;5;124m\"\u001b[39m\u001b[38;5;124mJames Balhoff\u001b[39m\u001b[38;5;124m\"\u001b[39m: \u001b[38;5;124m\"\u001b[39m\u001b[38;5;124mJames P. Balhoff\u001b[39m\u001b[38;5;124m\"\u001b[39m,\n\u001b[1;32m     21\u001b[0m }\n\u001b[1;32m     22\u001b[0m affiliation_rename \u001b[38;5;241m=\u001b[39m {\n\u001b[1;32m     23\u001b[0m     \u001b[38;5;124m\"\u001b[39m\u001b[38;5;124mSIB\u001b[39m\u001b[38;5;124m\"\u001b[39m: \u001b[38;5;124m\"\u001b[39m\u001b[38;5;124mSwiss Institute of Bioinformatics\u001b[39m\u001b[38;5;124m\"\u001b[39m,\n\u001b[1;32m     24\u001b[0m     \u001b[38;5;124m\"\u001b[39m\u001b[38;5;124mSIB Swiss Institute of Bioinformatics\u001b[39m\u001b[38;5;124m\"\u001b[39m: \u001b[38;5;124m\"\u001b[39m\u001b[38;5;124mSwiss Institute of Bioinformatics\u001b[39m\u001b[38;5;124m\"\u001b[39m,\n\u001b[0;32m   (...)\u001b[0m\n\u001b[1;32m    135\u001b[0m     \u001b[38;5;124m\"\u001b[39m\u001b[38;5;124mDivision of Microbiology and NCDs, ICMR-Regional Medical Research Centre, Bhubaneswar-23, Odisha, India\u001b[39m\u001b[38;5;124m\"\u001b[39m: \u001b[38;5;124m\"\u001b[39m\u001b[38;5;124mICMR-Regional Medical Research Centre\u001b[39m\u001b[38;5;124m\"\u001b[39m,\n\u001b[1;32m    136\u001b[0m }\n\u001b[1;32m    137\u001b[0m merge \u001b[38;5;241m=\u001b[39m {\n\u001b[1;32m    138\u001b[0m     (\u001b[38;5;241m100\u001b[39m, \u001b[38;5;241m215\u001b[39m),  \u001b[38;5;66;03m# chris hunter\u001b[39;00m\n\u001b[1;32m    139\u001b[0m     (\u001b[38;5;241m10\u001b[39m, \u001b[38;5;241m71\u001b[39m),  \u001b[38;5;66;03m# federica quaglia\u001b[39;00m\n\u001b[1;32m    140\u001b[0m     (\u001b[38;5;241m43\u001b[39m, \u001b[38;5;241m89\u001b[39m),  \u001b[38;5;66;03m# harry caufield\u001b[39;00m\n\u001b[1;32m    141\u001b[0m     (\u001b[38;5;241m63\u001b[39m, \u001b[38;5;241m324\u001b[39m),  \u001b[38;5;66;03m# jim balhoff\u001b[39;00m\n\u001b[1;32m    142\u001b[0m     (\u001b[38;5;241m288\u001b[39m, \u001b[38;5;241m77\u001b[39m),  \u001b[38;5;66;03m# Jana Sponarova\u001b[39;00m\n\u001b[1;32m    143\u001b[0m     (\u001b[38;5;241m107\u001b[39m, \u001b[38;5;241m34\u001b[39m),  \u001b[38;5;66;03m# jeet vora\u001b[39;00m\n\u001b[1;32m    144\u001b[0m     (\u001b[38;5;241m44\u001b[39m, \u001b[38;5;241m88\u001b[39m),  \u001b[38;5;66;03m# justin reese\u001b[39;00m\n\u001b[1;32m    145\u001b[0m     (\u001b[38;5;241m9\u001b[39m, \u001b[38;5;241m241\u001b[39m),  \u001b[38;5;66;03m# livia\u001b[39;00m\n\u001b[1;32m    146\u001b[0m     (\u001b[38;5;241m7\u001b[39m, \u001b[38;5;241m242\u001b[39m),  \u001b[38;5;66;03m# luana\u001b[39;00m\n\u001b[1;32m    147\u001b[0m     (\u001b[38;5;241m299\u001b[39m, \u001b[38;5;241m354\u001b[39m),  \u001b[38;5;66;03m# melissa haendel\u001b[39;00m\n\u001b[1;32m    148\u001b[0m     (\u001b[38;5;241m183\u001b[39m, \u001b[38;5;241m198\u001b[39m),  \u001b[38;5;66;03m# patrick masson\u001b[39;00m\n\u001b[1;32m    149\u001b[0m     (\u001b[38;5;241m38\u001b[39m, \u001b[38;5;241m346\u001b[39m),  \u001b[38;5;66;03m# raja\u001b[39;00m\n\u001b[1;32m    150\u001b[0m     (\u001b[38;5;241m110\u001b[39m, \u001b[38;5;241m346\u001b[39m),  \u001b[38;5;66;03m# raja\u001b[39;00m\n\u001b[1;32m    151\u001b[0m     (\u001b[38;5;241m332\u001b[39m, \u001b[38;5;241m369\u001b[39m), \u001b[38;5;66;03m# David OS\u001b[39;00m\n\u001b[1;32m    152\u001b[0m     (\u001b[38;5;241m395\u001b[39m, \u001b[38;5;241m1\u001b[39m), \u001b[38;5;66;03m# michael baudis\u001b[39;00m\n\u001b[1;32m    153\u001b[0m     (\u001b[38;5;241m379\u001b[39m, \u001b[38;5;241m326\u001b[39m), \u001b[38;5;66;03m# paola\u001b[39;00m\n\u001b[1;32m    154\u001b[0m     (\u001b[38;5;241m376\u001b[39m, \u001b[38;5;241m148\u001b[39m), \u001b[38;5;66;03m# rachael huntley\u001b[39;00m\n\u001b[0;32m--> 155\u001b[0m     \u001b[43m(\u001b[49m\u001b[38;5;241;43m204\u001b[39;49m\u001b[43m,\u001b[49m\u001b[43m \u001b[49m\u001b[38;5;241;43m391\u001b[39;49m\u001b[43m)\u001b[49m\u001b[43m \u001b[49m\u001b[38;5;66;43;03m#  Rahel Paloots\u001b[39;49;00m\n\u001b[1;32m    156\u001b[0m \u001b[43m    \u001b[49m\u001b[43m(\u001b[49m\u001b[38;5;241;43m374\u001b[39;49m\u001b[43m,\u001b[49m\u001b[43m \u001b[49m\u001b[38;5;241;43m328\u001b[39;49m\u001b[43m)\u001b[49m \u001b[38;5;66;03m# shawn tan\u001b[39;00m\n\u001b[1;32m    157\u001b[0m }\n\u001b[1;32m    158\u001b[0m merge \u001b[38;5;241m=\u001b[39m \u001b[38;5;28mdict\u001b[39m(merge)\n",
      "\u001b[0;31mTypeError\u001b[0m: 'tuple' object is not callable"
     ]
    }
   ],
   "source": [
    "person_rename = {\n",
    "    \"Cristina Casals Casas\": \"Cristina Casals-Casas\",\n",
    "    \"Chris Mungall\": \"Christopher J. Mungall\",\n",
    "    \"Cathy H Wu\": \"Cathy Wu\",\n",
    "    \"Frederic B. Bastian\": \"Frédéric B. Bastian\",\n",
    "    \"Frédéric Bastian\": \"Frédéric B. Bastian\",\n",
    "    \"Josie Burgin\": \"Josephine Burgin\",\n",
    "    \"Kristian Axelsen\": \"Kristian B. Axelsen\",\n",
    "    \"Lynn Schriml\": \"Lynn M Schriml\",\n",
    "    \"Mark Miller\": \"Mark A. Miller\",\n",
    "    \"Nicole A Vasilevsky\": \"Nicole A. Vasilevsky\",\n",
    "    \"Nicole Vasilevsky\": \"Nicole A. Vasilevsky\",\n",
    "    \"Paul D. Thomas\": \"Paul Denis Thomas\",\n",
    "    \"Paul Thomas\": \"Paul Denis Thomas\",\n",
    "    \"Rene Ranzinger\": \"René Ranzinger\",\n",
    "    \"Sujay Patil\": \"Sujay Sanjeev Patil\",\n",
    "    \"Susan Bello\": \"Susan M. Bello\",\n",
    "    \"Tarcisio Mendes De Farias\": \"Tarcisio Mendes de Farias\",\n",
    "    \"Chris Hunter\": \"Christopher Hunter\",\n",
    "    \"James Balhoff\": \"James P. Balhoff\",\n",
    "}\n",
    "affiliation_rename = {\n",
    "    \"SIB\": \"Swiss Institute of Bioinformatics\",\n",
    "    \"SIB Swiss Institute of Bioinformatics\": \"Swiss Institute of Bioinformatics\",\n",
    "    \"Swiss-Prot group, SIB Swiss Institute of Bioinformatics\": \"Swiss Institute of Bioinformatics\",\n",
    "    \"Paul Thomas Lab, University of Southern California (USC)\": \"University of Southern California\",\n",
    "    \"EMBL-European Bioinformatics Institute, Wellcome Genome Campus, Hinxton, Cambridge, UK\": \"EMBL-EBI\",\n",
    "    \"European Bioinformatics Institute (EMBL-EBI)\": \"EMBL-EBI\",\n",
    "    \"University of Lausanne (UNIL)\": \"University of Lausanne\",\n",
    "    \"Universite de Lausanne\": \"University of Lausanne\",\n",
    "    \"Complex Carbohydrate Research Center, The University of Georgia\": \"University of Georgia\",\n",
    "    \"Complex Carbohydrate Research Center, University of Georgia\": \"University of Georgia\",\n",
    "    \"The George Washington University\": \"George Washington University\",\n",
    "    \"LBNL\": \"Lawrence Berkeley National Laboratory\",\n",
    "    \"Environmental Genomics and Systems Biology Division, Lawrence Berkeley National Laboratory, Berkeley, CA, USA\": \"Lawrence Berkeley National Laboratory\",\n",
    "    \"Swiss Institute of Bioinformatics / Swiss-Prot group\": \"Swiss Institute of Bioinformatics\",\n",
    "    \"EMBL EBI\": \"EMBL-EBI\",\n",
    "    \"EMBL/EBI\": \"EMBL-EBI\",\n",
    "    \"EMBl/EBI\": \"EMBL-EBI\",\n",
    "    \"European Bioinformatics Institute\": \"EMBL-EBI\",\n",
    "    \"European Molecular Biology Laboratory, European Bioinformatics Institute (EMBL-EBI)\": \"EMBL-EBI\",\n",
    "    \"University of Luxembourg, Luxembourg Centre for Systems Biomedicine\": \"University of Luxembourg\",\n",
    "    \"Palacky University in Olomouc, Faculty of Science, Department of Physical Chemistry\": \"Palacký University Olomouc\",\n",
    "    \"Placky University Olomouc\": \"Palacký University Olomouc\",\n",
    "    \"Department of Biomedical Sciences, University of Padova\": \"University of Padova\",\n",
    "    \"European Molecular Biology Laboratory, European Bioinformatics Institute\": \"EMBL-EBI\",\n",
    "    \"University of Colorado, Anschutz Medical Campus\": \"University of Colorado\",\n",
    "    \"European Bioinformatics Institute (EMBL-EBI), Wellcome Genome Campus, Hinxton, CB10 1SD, UK.\": \"EMBL-EBI\",\n",
    "    \"European Bioinformatics Institute, EMBL-EBI\": \"EMBL-EBI\",\n",
    "    \"WormBase, Alliance of Genome Resources, California Institute of Technology\": \"California Institute of Technology\",\n",
    "    \"FlyBase, University of Cambridge\": \"University of Cambridge\",\n",
    "    \"Luddy School of Informatics, Computing, and Engineering, Indiana University, Bloomington, IN 47408, USA\": \"Indiana University\",\n",
    "    \"CBCB and CIS Department, University of Delaware\": \"University of Delaware\",\n",
    "    \"SIB and HES-SO (HEG)\": \"Swiss Institute of Bioinformatics\",\n",
    "    \"Keck School of Medicine, University of Southern California\": \"University of Southern California\",\n",
    "    \"EPFL\": \"Ecole Polytechnique Fédérale de Lausanne\",\n",
    "    \"Glycosciences Laboratory, Imperial College London\": \"Imperial College London\",\n",
    "    \"Department of Botany and Plant Pathology, Oregon State University\": \"Oregon State University\",\n",
    "    \"FlyBase/University Cambridge\": \"University of Cambridge\",\n",
    "    \"Swiss-Prot group, SIB Swiss Institute of Bioinformatics (SIB)\": \"Swiss Institute of Bioinformatics\",\n",
    "    \"EBI\": \"EMBL-EBI\",\n",
    "    \"Georgetown University Medical Center\": \"Georgetown University\",\n",
    "    \"Gene Ontology Consortium, Lawrence Berkeley National Laboratory\": \"Lawrence Berkeley National Laboratory\",\n",
    "    \"The University of Sydney, Sydney School of Veterinary Science\": \"University of Sydney\",\n",
    "    \"The Jackson Laboratory for Genomic Medicine\": \"Jackson Laboratory\",\n",
    "    \"The Jackson Laboratory\": \"Jackson Laboratory\",\n",
    "    \"Gene Ontology Consortium, University of Southern California\": \"University of Southern California\",\n",
    "    \"Healx UK\": \"Healx\",\n",
    "    \"Gene Ontology Consortium, The Jackson Laboratory\": \"Jackson Laboratory\",\n",
    "    \"Gene Ontology Consortium, Swiss Institute of Bioinformatics\": \"Swiss Institute of Bioinformatics\",\n",
    "    \"SGD/Stanford University\": \"Stanford University\",\n",
    "    \"GigaScience Journal, BGI-Hong Kong\": \"GigaScience\",\n",
    "    \"GigaScience Journal. BGI-Hong Kong\": \"GigaScience\",\n",
    "    \"BGI-Hong Kong, GigaScience team\": \"GigaScience\",\n",
    "    \"BGI, GigaScience team\": \"GigaScience\",\n",
    "    \"GigaScience, BGI\": \"GigaScience\",\n",
    "    \"European Nucleotide Archive, EMBL-EBI\": \"EMBL-EBI\",\n",
    "    \"ICBI, Georgetown University\": \"Georgetown University\",\n",
    "    \"Renaissance Computing Institute, University of North Carolina, Chapel Hill, NC US\": \"University of North Carolina\",\n",
    "    \"Mouse Genome Informatics, The Jackson Laboratory\": \"Jackson Laboratory\",\n",
    "    \"National Research Council, UniPD\": \"University of Padova\",\n",
    "    \"Gene Ontology Consortium, University of North Carolina\": \"University of North Carolina\",\n",
    "    \"School of Pharmaceutical Sciences, Siksha ‘O’ Anusandhan Deemed to be University, Bhubaneswar-03, Odisha, India\": \"Siksha O Anusandhan University\",\n",
    "    \"Lawrence Berkeley National Laboratory, Berkeley, CA\": \"Lawrence Berkeley National Laboratory\",\n",
    "    \"Department of Internal Medicine Translational Informatics Division, University of New Mexico, Albuquerque, NM, USA\": \"University of New Mexico\",\n",
    "    \"Swiss Institute of Bioinformatics. Swiss-Prot group\": \"Swiss Institute of Bioinformatics\",\n",
    "    \"University of New Mexico Health Sciences Center, Albuquerque, NM, USA\": \"University of New Mexico\",\n",
    "    \"University of Rome Tor Vergata\": \"Università di Tor Vergata\",\n",
    "    \"The Roslin Institute, University of Edinburgh\": \"University of Edinburgh\",\n",
    "    \"Edinburgh University\": \"University of Edinburgh\",\n",
    "    \"Jackson Laboratory for Genomic Medicine\": \"Jackson Laboratory\",\n",
    "    \"CIS Department, University of Delaware\": \"University of Delaware\",\n",
    "    \"CBCB, University of Delaware\": \"University of Delaware\",\n",
    "    \"TRANSLATIONAL AND INTEGRATIVE SCIENCES LAB\": \"University of Colorado\",\n",
    "    \"NEBION. Immunai company\": \"NEBION\",\n",
    "    \"NEBION, Immunai company\": \"NEBION\",\n",
    "    \"Reactome\": \"EMBL-EBI\",\n",
    "    \"Gene Ontology Consortium, California Institute of Technology\": \"California Institute of Technology\",\n",
    "    \"OICR\": \"Ontario Institute for Cancer Research\",\n",
    "    \"OICR (Canada), Federal University of Parana (Brazil)\": [\n",
    "        \"Ontario Institute for Cancer Research\",\n",
    "        \"Federal University of Parana\",\n",
    "    ],\n",
    "    \"University of Colorado Anschutz Medical Campus\": \"University of Colorado\",\n",
    "    \"Stanford Center for Biomedical Informatics Research, Stanford University, Stanford, CA, 94305 USA\": \"Stanford University\",\n",
    "    \"European Bioinformatics Institute (EMBL-EBI),Wellcome Genome Campus, Hinxton, CB10 1SD, UK.\": \"EMBL-EBI\",\n",
    "    \"European Bioinformatics Institute (EMBL-EBI), Wellcome Genome Campus, Hinxton, CB10 1SD, UK.\": \"EMBL-EBI\",\n",
    "    \"AstraZeneca UK\": \"AstraZeneca\",\n",
    "    \"Centre for Genomics Research, AstraZeneca\": \"AstraZeneca\",\n",
    "    \"Critical Path Institute, Tucson\": \"Critical Path Institute\",\n",
    "    \"Nanomedicine Laboratory, DBT-Institute of Life Sciences\": \"DBT Institute of Life Sciences\",\n",
    "    \"Nanomedicine Laboratory, DBT-Institute of Life Sciences, Bhubaneswar-23, Odisha, India\": \"DBT Institute of Life Sciences\",\n",
    "    \"Swiss Institute of Bioinformatics. Swiss-Prot group.\": \"Swiss Institute of Bioinformatics\",\n",
    "    \"Institute of Bio- and Geosciences, Bioinformatics (IBG-4), Forschungszentrum Jülich GmbH\": \"Forschungszentrum Jülich GmbH\",\n",
    "    \"Plant Genetic Resources Conservation Unit (PGRCU) of the National Plant Germplasm System (NPGS), USDA ARS\": \"United States Department of Agriculture\",\n",
    "    \"National Clonal Germplasm Repository (NCGR), USDA ARS\": \"United States Department of Agriculture\",\n",
    "    \"Swiss Institute of Bioinformatics (SIB), University of Lausanne (UNIL)\": [\n",
    "        \"Swiss Institute of Bioinformatics\",\n",
    "        \"University of Lausanne\",\n",
    "    ],\n",
    "    \"University of Lausanne (UNIL), Swiss Institute of Bioinformatics (SIB)\": [\n",
    "        \"Swiss Institute of Bioinformatics\",\n",
    "        \"University of Lausanne\",\n",
    "    ],\n",
    "    \"SIB Swiss Institute of Bioinformatics, University of Lausanne\": [\n",
    "        \"Swiss Institute of Bioinformatics\",\n",
    "        \"University of Lausanne\",\n",
    "    ],\n",
    "    \"Cold Spring Harbor Laboratory & USDA ARS NEA\": [\n",
    "        \"Cold Spring Harbor Laboratory\",\n",
    "        \"United States Department of Agriculture\",\n",
    "    ],\n",
    "    \"ZHAW\": \"Zurich University of Applied Sciences\",\n",
    "    \"National Clonal Germplasm Repository (NCGR), USDA ARS\": \"United States Department of Agriculture\",\n",
    "    \"Division of Microbiology and NCDs, ICMR-Regional Medical Research Centre, Bhubaneswar-23, Odisha, India\": \"ICMR-Regional Medical Research Centre\",\n",
    "}\n",
    "merge = {\n",
    "    (100, 215),  # chris hunter\n",
    "    (10, 71),  # federica quaglia\n",
    "    (43, 89),  # harry caufield\n",
    "    (63, 324),  # jim balhoff\n",
    "    (288, 77),  # Jana Sponarova\n",
    "    (107, 34),  # jeet vora\n",
    "    (44, 88),  # justin reese\n",
    "    (9, 241),  # livia\n",
    "    (7, 242),  # luana\n",
    "    (299, 354),  # melissa haendel\n",
    "    (183, 198),  # patrick masson\n",
    "    (38, 346),  # raja\n",
    "    (110, 346),  # raja\n",
    "    (332, 369), # David OS\n",
    "    (395, 1), # michael baudis\n",
    "    (379, 326), # paola\n",
    "    (376, 148), # rachael huntley\n",
    "    (204, 391) #  Rahel Paloots\n",
    "    (374, 328) # shawn tan\n",
    "}\n",
    "merge = dict(merge)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "22b55474",
   "metadata": {},
   "outputs": [],
   "source": [
    "directory = here.parent.joinpath(\"data\")\n",
    "\n",
    "resources = Path()\n",
    "\n",
    "df = pd.read_csv(directory / \"author.csv\")\n",
    "\n",
    "df = df[df[\"first name\"].notna()]\n",
    "df[\"person #\"] = df[\"person #\"].map(lambda x: merge.get(x, x))\n",
    "df[\"affiliation\"] = df[\"affiliation\"].map(lambda x: x.strip().replace(\"  \", \" \"))\n",
    "df[\"affiliation\"] = df[\"affiliation\"].map(\n",
    "    lambda x: affiliation_rename.get(x.strip(), x.strip())\n",
    ")\n",
    "df = df[~df[\"last name\"].str.contains(\"Consortium\")]\n",
    "df = df[~df[\"last name\"].str.contains(\"consortium\")]\n",
    "df[\"name\"] = [\n",
    "    f\"{first} {last}\" for first, last in df[[\"first name\", \"last name\"]].values\n",
    "]\n",
    "df[\"name\"] = df[\"name\"].map(lambda x: person_rename.get(x.strip(), x.strip()))\n",
    "del df[\"first name\"]\n",
    "del df[\"last name\"]\n",
    "df[\"email\"] = df[\"email\"].str.lower()\n",
    "\n",
    "df.head()\n",
    "# pd.set_option('display.max_rows', None)\n",
    "# df[\"affiliation\"].value_counts().to_frame()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "1389ef66",
   "metadata": {},
   "outputs": [],
   "source": [
    "if True or not authors_path.is_file():\n",
    "    # don't overwrite this after curation has started\n",
    "    cols = [\"person #\", \"name\", \"email\", \"country\", \"affiliation\"]\n",
    "    sdf = df[cols].sort_values(\"name\").drop_duplicates([\"person #\", \"name\"])\n",
    "    sdf.to_csv(authors_path, sep=\"\\t\", index=False)\n",
    "\n",
    "affiliations_counter = Counter(\n",
    "    x for a in df[\"affiliation\"] for x in ([a] if isinstance(a, str) else a)\n",
    ")\n",
    "if True or not affiliations_path.is_file():\n",
    "    # don't overwrite this after curation has started\n",
    "    with affiliations_path.open(\"w\") as file:\n",
    "        print(\"name\", \"ror\", \"wikidata_ids\", file=file, sep=\"\\t\")\n",
    "        for affiliation in sorted(affiliations_counter):\n",
    "            print(affiliation, \"\", \"\", sep=\"\\t\", file=file)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "e6313fe5",
   "metadata": {},
   "outputs": [],
   "source": [
    "n_authors = df[\"person #\"].nunique()\n",
    "n_authors"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "4d3f0c35",
   "metadata": {},
   "outputs": [],
   "source": [
    "n_abstracts = df[\"submission #\"].nunique()\n",
    "n_abstracts"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "18aad8e7",
   "metadata": {},
   "outputs": [],
   "source": [
    "n_affiliations = len(affiliations_counter)\n",
    "n_affiliations"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "d76d9a64",
   "metadata": {},
   "outputs": [],
   "source": [
    "affiliation_to_papers = defaultdict(set)\n",
    "for paper, affiliations in df[[\"submission #\", \"affiliation\"]].values:\n",
    "    for affiliation in (\n",
    "        [affiliations] if isinstance(affiliations, str) else affiliations\n",
    "    ):\n",
    "        affiliation_to_papers[affiliation].add(paper)\n",
    "\n",
    "affiliation_to_paper_count = Counter(\n",
    "    {a: len(p) for a, p in affiliation_to_papers.items()}\n",
    ")\n",
    "affiliation_to_paper_count.most_common(10)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "95186334",
   "metadata": {
    "scrolled": false
   },
   "outputs": [],
   "source": [
    "fix, ((lax, rax), (blax, brax), (bblax, bbrax)) = plt.subplots(3, 2, figsize=(8.5, 7))\n",
    "\n",
    "sns.countplot(\n",
    "    data=df.groupby(\"submission #\").count()[\"person #\"].to_frame(),\n",
    "    x=\"person #\",\n",
    "    ax=lax,\n",
    ")\n",
    "lax.set_xlabel(\"# Authors\")\n",
    "lax.set_ylabel(\"# Abstracts with # Authors\")\n",
    "lax.set_title(\"Author Distribution\")\n",
    "\n",
    "sns.countplot(\n",
    "    data=df[[\"person #\", \"country\"]].drop_duplicates(),\n",
    "    y=\"country\",\n",
    "    order=df[[\"person #\", \"country\"]].drop_duplicates()[\"country\"].value_counts().index,\n",
    "    ax=rax,\n",
    ")\n",
    "rax.set_ylabel(\"\")\n",
    "rax.set_title(\"Author Locations\")\n",
    "rax.set_xscale(\"log\")\n",
    "\n",
    "sns.countplot(\n",
    "    data=df.groupby(\"person #\")\n",
    "    .count()[\"submission #\"]\n",
    "    .sort_values(ascending=False)\n",
    "    .to_frame(),\n",
    "    x=\"submission #\",\n",
    "    ax=blax,\n",
    ")\n",
    "blax.set_yscale(\"log\")\n",
    "blax.set_xlabel(\"# Abstracts Authored\")\n",
    "blax.set_ylabel(\"# Authors\")\n",
    "blax.set_title(\"Author Prolificness\")\n",
    "\n",
    "plt.suptitle(\n",
    "    f\"Summaries of {n_abstracts} Abstracts from {n_authors} Authors at {n_affiliations} Affiliations\"\n",
    ")\n",
    "plt.tight_layout()\n",
    "plt.savefig(\"/Users/cthoyt/Desktop/biocuration2023_abstracts_summary.png\", dpi=350)\n",
    "plt.show()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "92eae8f9",
   "metadata": {},
   "outputs": [],
   "source": [
    "fig, (lax, rax) = plt.subplots(1, 2, figsize=(8.5, 4.5))\n",
    "\n",
    "n = 20\n",
    "y, x = zip(*affiliations_counter.most_common(n))\n",
    "sns.barplot(\n",
    "    y=list(y),\n",
    "    x=list(x),\n",
    "    ax=lax,\n",
    ")\n",
    "lax.set_xlabel(\"Number of Authors\")\n",
    "lax.set_title(f\"Top {n} Submitting Affiliations\")\n",
    "\n",
    "\n",
    "affiliation_multiple_submission_rows = list(\n",
    "    (name, count)\n",
    "    for name, count in affiliation_to_paper_count.most_common(n)\n",
    "    if count > 1\n",
    ")\n",
    "y, x = zip(*affiliation_multiple_submission_rows)\n",
    "sns.barplot(\n",
    "    y=list(y),\n",
    "    x=list(x),\n",
    "    ax=rax,\n",
    ")\n",
    "rax.set_xlabel(\"Number of Papers\")\n",
    "rax.set_title(f\"Top {n} Submitting Affiliations\")\n",
    "\n",
    "\n",
    "plt.suptitle(f\"Summaries of {n_affiliations} Affiliations\")\n",
    "plt.tight_layout()\n",
    "plt.savefig(\"/Users/cthoyt/Desktop/biocuration2023_abstracts_summary.png\", dpi=350)\n",
    "plt.show()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "ee8e6e44",
   "metadata": {},
   "outputs": [],
   "source": [
    "print(\n",
    "    tabulate(\n",
    "        affiliation_multiple_submission_rows,\n",
    "        tablefmt=\"github\",\n",
    "        headers=[\"Institution\", \"Submissions\"],\n",
    "    )\n",
    ")"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "a715c5e4",
   "metadata": {},
   "outputs": [],
   "source": [
    "names = dict(df[[\"person #\", \"name\"]].drop_duplicates().values)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "9b6a710b",
   "metadata": {},
   "outputs": [],
   "source": [
    "print(\n",
    "    tabulate(\n",
    "        [(names[n], count) for n, count in Counter(df[\"person #\"]).most_common(10)],\n",
    "        tablefmt=\"github\",\n",
    "        headers=[\"Name\", \"Count\"],\n",
    "    )\n",
    ")"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "26641f7b",
   "metadata": {},
   "outputs": [],
   "source": [
    "submissions_path = directory.joinpath(\"submission.csv\")\n",
    "submissions_df = pd.read_csv(submissions_path)\n",
    "del submissions_df[\"notified\"]\n",
    "del submissions_df[\"reviews sent\"]\n",
    "del submissions_df[\"last updated\"]\n",
    "\n",
    "def _map(s):\n",
    "    s=s.lower()\n",
    "    if \"lightning\" in s:\n",
    "        return \"lightning\"\n",
    "    if \"oral\" in s:\n",
    "        return \"oral\"\n",
    "    elif \"poster\" in s:\n",
    "        return \"poster\"\n",
    "    elif \"reject\" in s:\n",
    "        return \"reject\"\n",
    "    raise ValueError(s)\n",
    "\n",
    "submissions_df[\"decision\"] =submissions_df[\"decision\"].map(_map)\n",
    "\n",
    "submissions_df.head()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "095596b3",
   "metadata": {},
   "outputs": [],
   "source": [
    "data_directory = here.parent.joinpath(\"_data\")\n",
    "import yaml\n",
    "\n",
    "def _process_authors(s):\n",
    "    p = [a.strip() for a in authors.split(\",\")]\n",
    "    if \" and \" in p[-1]:\n",
    "        return [*p[:-1], *(a.strip() for a in p[-1].split(\" and \"))]\n",
    "    return p\n",
    "    \n",
    "    \n",
    "for decision, sdf in submissions_df.groupby(\"decision\"):\n",
    "    if decision == \"reject\":\n",
    "        continue\n",
    "    d = []\n",
    "    for _n, title, authors, _, _, keywords, _, abstract in sdf.values:\n",
    "        d.append({\n",
    "            \"title\": title,\n",
    "            \"authors\": [\n",
    "                {\"label\": a, \"series_ordinal\": i}\n",
    "                for i, a in enumerate(_process_authors(authors), start=1)\n",
    "            ],\n",
    "            \"keywords\": [k.strip() for k in keywords.split(\"\\n\")],\n",
    "            \"abstract\": abstract.strip()\n",
    "        })\n",
    "    with data_directory.joinpath(decision).with_suffix(\".yml\").open(\"w\") as file:\n",
    "        yaml.safe_dump(d, file, allow_unicode=True)"
   ]
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "Python 3 (ipykernel)",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.11.0"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 5
}
